portfolio/src/main/webapp/community.js

```diff
@@ -0,0 +1,40 @@
+function renderComments() {
+let quantity = document.getElementById("quantity").value;
+  fetch('/comments?quantity=' + quantity).then(response => response.json()).then((conversation) => {
+    console.log(conversation);
+
+    let commentSection = document.getElementById('submitted-comments');
+    while (commentSection.lastChild) {
+        commentSection.removeChild(commentSection.lastChild);
+    }
+
+    conversation.forEach((comment) => {
+        createComment(comment);
+    });
+  });
+}
+
+function createComment(comment) {
+    const commentSection = document.getElementById('submitted-comments');
+    const commentDivElement = document.createElement('div');
+    const usernameElement = document.createElement('p');
+    const commentElement = document.createElement('p');
+    const exitButtonElement = document.createElement('button');
+
+    usernameElement.innerHTML = comment.username + " says:";
+    commentElement.innerHTML = comment.commentText;
+    exitButtonElement.innerHTML = "×";
```

Comment:
We want to avoid directly manipulating the innerHTML as much as we can.

Try "textNode" with "appendChild" instead:
https://www.w3schools.com/jsref/met_node_appendchild.asp

Reply:
Resolved